beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/utils/BeekeeperApiTestClient.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (C) 2019-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.integration.utils;
+
+import static java.net.http.HttpRequest.newBuilder;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+
+
+public class BeekeeperApiTestClient {
+  
+  private static final String PREFIX = "/api/v1";
+  private static final String DATABASE_AND_TABLE = "/database/some_database/table/some_table";
```

Comment:
Why do you need to have these broken down into sections? They are only used to build the `metadataUrl` once.

Reply:
Because in the future if we add other endpoints, we might need to use "/api/v1" but not "/database/some_database/table/some_table". Either way I've renamed a lot of stuff in this class so u can take another look:)

Reply:
Fair enough. I'll take a look. Thanks.